src/github/queries.gql

```diff
@@ -112,7 +112,7 @@ fragment ReviewComment on PullRequestReviewComment {
 	state
 	pullRequestReview { databaseId }

```

Comment:
looks like 	`viewerCanDelete` and `viewerCanUpdate` are on `IssueComment`, but also need to be added here

Reply:
resolved.